Week-6/ES6Module/calculator.js

```diff
@@ -0,0 +1,9 @@
+// Define the add function
+export function add(a, b) {
+    return a + b;
+}
+
+// Define the subtract function
+export function subtract(a, b) {
```

Comment:
Consider including a default export in your modules if there is a primary functionality that is expected to be used frequently.